modules/circe/src/main/scala/io/chrisdavenport/fuuid/circe/package.scala

```diff
@@ -0,0 +1,11 @@
+package io.chrisdavenport.fuuid
+
+import cats.syntax.show._
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+
+package object circe {
+  implicit val encoder: Encoder[FUUID] = Encoder.instance(_.show.asJson)
```

Comment:
Just thought of this, due to namespace importing these should likely be named `fuuidEncoder`/`fuuidDecoder` so that we're not playing on other peoples namespaces.

Reply:
yeap, true. thank you!